docker/entrypoint.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
-mkdir -p  /var/log/go-dmux
+mkdir -p  /var/log/flipkart/go-dmux
+
+#if [[ -z ${LOG_DIR} ]]; then
+#  export LOG_DIR="/var/log/go-dmux"
+#  mkdir -p ${LOG_DIR}
+#fi
 sleep 10
-/app/go-dmux /app/config/conf.json > /var/log/go-dmux/stdout.log 2> /var/log/go-dmux/stderr.log
+/app/go-dmux /app/config/conf.json
```

Comment:
```suggestion
# arguments for the app will be passed at runtime
exec /app/go-dmux "$@"
```

Reply:
ok .. then this command 'exec /app/go-dmux $@' shouldn't be in entrypoint.sh right.. and should be in the deployment yaml?

Reply:
Ok I think I got you know .. we can pass entrypoint args to k8s deployment yaml as mentioned here ..  https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#container-v1-core